orca-pipelinetemplate/src/test/resources/gate.yml

```diff
@@ -0,0 +1,150 @@
+#document netflix-conventions
+clusters:
+  netflix-conventions:
+    requires:
+      membership:
```

Comment:
this isn't really effective because list gets clobbered by an override instead of merged.  could restructure to Map<String, Object>, but there isn't really anything I want to put in the actual object payload